addons/survey/views/survey_templates_statistics.xml

```diff
@@ -70,70 +63,51 @@
-                    <span class="badge badge-success ml-3"><span t-esc="question_data['right_inputs_count']"></span> Correct</span>
+                <!-- Question info -->
+                <span class="badge badge-info" t-field='question.question_type'/>
+                <t t-if="question.question_type == 'matrix'">
+                    <span class="badge badge-info ml-2" t-field='question.matrix_subtype'/>
                 </t>
-                <t t-if="question.question_type in ['simple_choice', 'multiple_choice']">
-                    <span class="badge badge-warning ml-1" t-if="question.question_type == 'multiple_choice'">
-                        <span t-esc="question_data['partial_inputs_count']"></span> Partial
-                    </span>
+                <!-- Scoring info -->
+                <t t-if="question.is_scored_question">
+                    <t t-if="question.question_type in ['simple_choice', 'multiple_choice'] or question.is_scored_question">
+                        <span class="badge badge-success ml-3"><span t-esc="question_data['right_inputs_count']"/> Correct</span>
+                    </t>
+                    <t t-if="question.question_type in ['simple_choice', 'multiple_choice']">
+                        <span class="badge badge-warning ml-1" t-if="question.question_type == 'multiple_choice'">
+                            <span t-esc="question_data['partial_inputs_count']"></span> Partial
+                        </span>
+                    </t>
                 </t>
-            </t>
-            <!-- Inputs info -->
-            <span class="badge badge-info ml-3"><span t-esc="len(question_data['answer_input_done_ids'])"></span> Answered</span>
-            <span class="badge badge-info ml-1"><span t-esc="len(question_data['answer_input_skipped_ids'])"></span> Skipped</span>
-        </div>
+                <!-- Inputs info -->
+                <span class="badge badge-info ml-3"><span t-esc="len(question_data['answer_input_done_ids'])"></span> Answered</span>
+                <span class="badge badge-info ml-1"><span t-esc="len(question_data['answer_input_skipped_ids'])"></span> Skipped</span>
+            </div>
 
-        <!-- Question Description -->
-        <div class="ml-3 text-muted" t-field="question.description"/>
-            <t t-if="question.question_type in ['text_box', 'char_box', 'datetime']">
-                <t t-call="survey.question_result_text"/>
+            <!-- Question Description -->
+            <div class="ml-3 text-muted" t-field="question.description"/>
+
+            <!-- Answers -->
+            <t t-set="question_answered" t-value="question_data['answer_input_done_ids']"/>
+            <div t-if="not question_answered" class="alert alert-secondary m-3" role="alert">
+                No one answered this question.
+            </div>
+
+            <t t-set="table_active" t-value="'active' if not question_answered else ''"/>
```

Comment:
I think I would avoid a variable and re-specify the condition every time in the attf since it's a one-liner.

Helps when reading the code, otherwise you need to just to the definition of this to know what it's based on.